src/main/java/net/snowflake/client/core/SFBaseSession.java

```diff
@@ -399,6 +410,7 @@ public HttpClientSettingsKey getHttpClientKey() throws SnowflakeSQLException {
       String nonProxyHosts = systemGetProperty("http.nonProxyHosts");
       // log the JVM parameters that are being used
       if (httpUseProxy) {
+        logger.debug("Using JVM parameters for proxy setup");
```

Comment:
can we join this log with the one below?

Reply:
Done.